src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -110,6 +113,11 @@ const AudioInputVFControl: React.FC<Props> = ({
             setIsLoading(true);
             const receivedDevice = audioInputSelectionToDevice(device.deviceId);
             const currentDevice = await addVoiceFocus(receivedDevice);
+
+            console.info('ObserveMeetingAudio:', audioVideo);
+            await (currentDevice as VoiceFocusTransformDevice).observeMeetingAudio(audioVideo as AudioVideoFacade);
+            console.info('ObserveMeetingAudio:', audioVideo);
```

Comment:
Do we need to keep these two `console.info`?

Reply:
They should be removed.

Reply:
removed.